core/chains/evm/logpoller/models.go

```diff
@@ -32,3 +32,11 @@ type Log struct {
 	Data        []byte
 	CreatedAt   time.Time
 }
+
+func (l *Log) GetTopics() []common.Hash {
```

Comment:
nit: can be value receiver

Reply:
He had value and I recommended switching to pointer. Isn't pointer more efficient here? Since value would involve copying all the other unrelated struct fields.

Reply:
ah the old efficiency vs safety tradeoff. I just prefer the safety since perf difference is negligible here

Reply:
I agree that in this particular case, either option is not a big deal. Will leave as is for now.